### PR TITLE
Enable extended registration form for Lagunita

### PR DESCRIPTION
**Pull request by stvstnfrd**
Technically, we _could_ disable this for SUclass, but since we don't
even load the registration page there, it seems unnecessary.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
Ping: @dcadams

**Comment by dcadams** [comment]
Why are you not putting this into a stanford/lagunita/group_vars file?

**Comment by stvstnfrd** [comment]
> Why are you not putting this into a stanford/lagunita/group_vars file?

We want this to apply everywhere, not just lagunita & stage, including devstack (which doesn't pull from `stanford/*/group_vars`.
The changes are made to the "Stanford-specific" section of this shared file.

**Comment by dcadams** [comment]
:+1: assuming we won't end up with data in Marketo from a non-lagunita instance.

**Comment by stvstnfrd** [comment]
> 👍 assuming we won't end up with data in Marketo from a non-lagunita instance.

Yeah, we won't be setting up a `cron`/Jenkins job for any other instance, so no data will ever be sent to Marketo.

The only impact here is that the data table will be created (and maybe populated?) on other instances;
if we want to avoid that, we can disable this setting for them (eg: SUclass).

FWIW: On Lagunita, we _do_ have the (old) `cme_registration` table (it's empty), but not the newer table; both tables _do_ exist on stage.

File: stanford/suclass/group_vars/all [hunk 1]
```diff
@@ -15,6 +17,7 @@ EDXAPP_LMS_BASE: 'suclass.stanford.edu'
 EDXAPP_LMS_BASE_SCHEME: 'https'
 EDXAPP_LMS_NGINX_PORT: 80
 EDXAPP_PREVIEW_LMS_BASE: "preview.{{ EDXAPP_LMS_SITE_NAME }}"
+EDXAPP_REGISTRATION_EXTENSION_FORM: ''
```
Comment (stvstnfrd) [inline stanford/suclass/group_vars/all]: @dcadams I went ahead and disabled this anyway on SUclass;
then we don't need to worry about SUclass migrations
and the extra table/data.

**Pull request merged**